sdk/tables/Azure.Data.Tables/README.md

```diff
@@ -13,7 +13,7 @@ Azure Cosmos DB provides a Table API for applications that are written for Azure
 
 The Azure Tables client library can seamlessly target either Azure Table storage or Azure Cosmos DB table service endpoints with no code changes.
 
-[Source code][table_client_src] | [Package (NuGet)][table_client_nuget_package] | [API reference documentation][api_reference] | [Samples][table_client_samples]
+[Source code][table_client_src] | [Package (NuGet)][table_client_nuget_package] | [API reference documentation][api_reference] | [Samples][table_client_samples] | [Change Log][table_change_log]
```

Comment:
Is this something we are doing to all clients?

Reply:
Not necessarily, but we could. This was a specific ask from the Cosmos team when David Berry presented his updated .NET tables example webapp to them.

Reply:
I think we should consider making this standard